tmt/steps/provision/testcloud.py

```diff
@@ -235,6 +235,19 @@ def guest(self):
         """ Return the provisioned guest """
         return self._guest
 
+    @classmethod
+    def clean_images(cls, clean, dry):
```

Comment:
I don't really like passing `clean` and `dry` to this method but I haven't found a way to get option/log inside a classmethod. @psss am I missing a way of doing that? I think this method has to be a class method because it doesn't make much sense to instantiate an instance.

Reply:
For the class methods there could be used the `Class._opt()` method to get the options from the stored command line context. Did not find a better way than to have a separate method for this.